overlays/bootstrap.nix

```diff
@@ -365,6 +365,27 @@ in {
 
                 ghc-patches = ghc-patches "8.10.2";
             };
+            # ghc 8.10.2 with patches needed by plutus
+            ghc8102-plutus = final.callPackage ../compiler/ghc {
+                extra-passthru = { buildGHC = final.buildPackages.haskell-nix.compiler.ghc8102; };
+
+                bootPkgs = bootPkgs // {
+                  ghc = final.buildPackages.buildPackages.haskell-nix.compiler.ghc884;
+                };
+                inherit sphinx installDeps;
+
+                buildLlvmPackages = final.buildPackages.llvmPackages_9;
+                llvmPackages = final.llvmPackages_9;
+
+                src-spec = rec {
+                    version = "8.10.2";
+                    url = "https://downloads.haskell.org/~ghc/${version}/ghc-${version}-src.tar.xz";
+                    sha256 = "02w8n085bw38vyp694j0lfk5wcnwkdaj7hhp0saj71x74533lmww";
+                };
+
+                ghc-patches = ghc-patches "8.10.2"
+                 ++ [ ./patches/ghc/ghc-8.10-ubxt.patch ];
```

Comment:
Could we just sneak this into our patches list for 8.10? Or is that a bit risky?

Reply:
It was a little risky since it turned type check error telling use that unboxed tuples would not work into a runtime crash. 
@luite improved the patch so it is less risky (unboxed tuples should now work not just type check) and #825 has now been merged into haskell.nix master for ghc 8.10.2.